File: r/FizzBuzz.r
iterations <- 1:100

for (i in iterations) {

    say <- ''

    if (i %% 3 == 0) {
        say <- paste(say, 'Fizz', sep='')
    }

    if (i %% 5 == 0) {
        say <- paste(say, 'Buzz', sep='')
    }

    if (identical(say, '')) {
        print(i)
    } else {
        print(say)
    }

}

File: dart/FizzBuzz.dart
void main() {
  
  int iterations = 100;
  
  for (int i = 1; i <= iterations; i++) {
    String say = '';
    
    if (i % 3 == 0){
      say = '$say''Fizz';
    }
    
    if (i % 5 == 0){
      say = '$say''Buzz';
    }
    
    if (say == ''){
      print('$i');
    } else {
      print(say);
    }
    
  }
}
